api/auth.py

```diff
@@ -0,0 +1,22 @@
+from drf_yasg.utils import swagger_auto_schema
+from rest_framework import serializers
+
+from rest_framework_simplejwt.views import (
+    TokenObtainPairView,
+    TokenRefreshView,
+    TokenVerifyView
+)
+
+
+class TokenSerializer(serializers.Serializer):
+    refresh = serializers.CharField()
+    access = serializers.CharField()
+
+class TokenErrorSerializer(serializers.Serializer):
+    details = serializers.CharField()
+
+# Override method to fix swagger schema
+class JWTAuthToken(TokenObtainPairView):
```

Comment:
@xitij2000 I had to override the method and provide correct response so that the auto generated client could work.

Reply:
We might want to look into doing this through the customisation support provided by swagger-codegen.